### DISCUSSION TITLE
"Error Connection Timed Out" when merging checkpoints

### DISCUSSION THREAD

**Discussion by zylervega** [TheLastBen/fast-stable-diffusion#261] [Q&A]
![image](https://user-images.githubusercontent.com/100597464/198327763-5f3cff86-da54-4262-8186-0d1ceefc8a6e.png)

1. run "fast_stable_diffusion_AUTOMATIC1111.ipynb" pointed at a directory with several models
2. load local page
3. attempt to merge 3 models
4. wait a minute or two
5. this happens:
![image](https://user-images.githubusercontent.com/100597464/198327013-4678cfc1-5d69-4054-bb7c-09b7072d013c.png)

I am running these as google colabs.  I have google colab pro.
Is there anything I can do to prevent this error?

After it times out, reloading produces a 504 error.

I am always able to rerun the "Start stable-diffusion" cell to spawn a new local site which will work long enough to allow me to start over with the same results.

**Comment by zylervega** [comment]
Am I misinterpreting what merging checkpoints does?  It is my understanding that I can merge different _models_ using this feature.
Is that incorrect?

**Reply by TheLastBen** [reply]
The RAM is insufficient

**Reply by zylervega** [reply]
Ah, thanks.  I'll try smaller models to confirm.

**Reply by Vailoiro** [reply]
i'm having the same issue, "Something went wrong Connection errored out." Is it just not enough RAM?

**Reply by ichhit-ai** [reply]
> The RAM is insufficient

how much ram is enough

**Comment by THX1139b** [comment]
I now get this error when upscaling above 2000x3000 pixels using Colab Pro with high RAM setting. This has not been a problem for me before. Hoping that it's just a temporary thing as I need images for print. I do realize that this is pushing things a bit but I happily made poster sized upscales of some line art illustrations before.

**Reply by TheLastBen** [reply]
try runpod fast stable diffusion for faster sd

**Comment by Vailoiro** [comment]
I also have the same problem, when i'm changing between models, is it not enough RAM?

**Reply by TheLastBen** [reply]
very likely a colab limitation, you can use Runpod or Paperspace to perform the merging as they offer 30gb+ RAM
